src/tests/JIT/Regression/JitBlue/Runtime_68049/Runtime_68049_1.cs

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Generated by Fuzzlyn v1.5 on 2022-04-13 11:57:44
+// Run on X64 Windows
+// Seed: 15524766127402705757
+// Reduced from 138.3 KiB to 1.6 KiB in 00:02:03
+// Debug: Outputs 1
+// Release: Outputs 254
+
+public struct S0
+{
+    public short F0;
+    public int F3;
+    public byte M29(ref sbyte[] arg0)
+    {
+        if (this.F0 > this.F3)
+        {
+            this.F0 = (short)~this.F0;
+            sbyte[, ] var0 = new sbyte[, ]{{1}};
+        }
+
+        return Runtime_68049_1.M31();
+    }
+}
+
+public class Runtime_68049_1
+{
+    public static byte s_result;
+    public static IRuntime s_rt;
+    public static short[] s_2 = new short[]{0};
+    public static sbyte[][] s_8 = new sbyte[][]{new sbyte[]{1}};
+    public static byte s_9 = 28;
+    public static int Main()
+    {
+        CollectibleALC alc = new CollectibleALC();
+        System.Reflection.Assembly asm = alc.LoadFromAssemblyPath(System.Reflection.Assembly.GetExecutingAssembly().Location);
+        System.Reflection.MethodInfo mi = asm.GetType(typeof(Runtime_68049_1).FullName).GetMethod(nameof(MainInner));
+        System.Type runtimeTy = asm.GetType(typeof(Runtime).FullName);
+        mi.Invoke(null, new object[]{System.Activator.CreateInstance(runtimeTy)});
+        return (int) s_result;
```

Comment:
I would remove this test. It requires loading the assembly from file which means it has to be disabled in various configurations in issues.targets. Also I think the `s_result` is being set in the other instance of the type (the one loaded in the other ALC), not in this one, so it will always fail. Probably easier to just rely on the other test.

Reply:
Ok.